<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiManager</class>
 <widget class="QWidget" name="GuiManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuiManager</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>240</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="canvas">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>241</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnminimumwidth="100,0">
    <item row="4" column="1">
     <widget class="QLineEdit" name="decelerationLineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="positionxLineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="accelerationLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="velocityLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>deceleration:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>jerk:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="jerkLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>positionx:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>velocity:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>acceleration:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>positiony:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="positionyLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addCmdButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>AddNewCommand</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>460</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>currentvelocity:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="currentVelocityLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
